code/datums/preferences.dm

```diff
@@ -1309,23 +1309,30 @@ datum/preferences
 			HTML += "</td>"
 
 		HTML += "<td valign='top' class='antagprefs'>"
+		if (user?.client?.player.get_rounds_participated() < TEAM_BASED_ROUND_REQUIREMENT && !jobban_isbanned(user, "Syndicate"))
```

Comment:
What's with the additional jobban check? This is done later.

Reply:
Done to prevent the double HTML message (round req and ban message)

Reply:
either have both or save the result of this jobban check so we don't perform it twice

Reply:
Done